src/module.ts

```diff
@@ -44,12 +86,14 @@ export default defineNuxtModule<ModuleOptions>({
       basic: true,
       utilities: false,
     },
+    config: {},
   },
   async setup(options, nuxt) {
     nuxt.options.build.transpile.push(runtimeDir)
     nuxt.options.build.transpile.push(/@ionic/, /@stencil/)
+    nuxt.options.runtimeConfig.ionic = options.config
```

Comment:
This will only work on the server as only `runtimeConfig.public.*` properties are forwarded to the client. Are we expecting these properties to change once built, or can we include them in the build?

Reply:
Aaa, ya it should be on public  
I think better to include them in build, will be useful for user to be able check actives config, wdyt?

Reply:
If we include them in build, we could directly add a template with `addTemplate` rather than including them in payload.

Reply:
Let's keep with current change then, for now

Reply:
do you think users need to be able to view what these options are by calling `useRuntimeConfig()`?

Reply:
I've gone ahead moved it to a template, but please do let me know if you think there is a use-case for runtime config.